helm-charts/falcon-kac/values.schema.json

```diff
@@ -259,6 +259,16 @@
                     ]
                 }
             }
+        },
+        "resourceQuota": {
+            "type": "object",
+            "properties": {
+                "pods": {
+                    "type": "integer",
+                    "default": "2",
+                    "minimum": 2
```

Comment:
This can be an integer or a string and should be handled for either use-case.

Reply:
Thanks for the review. Are you saying something like? 
```yaml
# values.schema.json file
        "resourceQuota": {
            "type": "object",
            "properties": {
                "pods": {
                    "oneOf": [
                        {
                            "type": "integer",
                            "default": "2",
                            "minimum": 2
                        },
                        {
                            "type": "string",
                            "default": "2"
                        }
                    ]
                }
            }
        }

# resourcequota.yaml
spec:
  hard:
    pods: {{ .Values.resourceQuota.pods | quote }}
```

Reply:
Sorry, I missed this question, but yes, something like that were there is the user of oneOf.